src/AddComment.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from "react";
+import { postComment } from "./utils/api";
+
+export function AddComment({ setComments, article_id }) {
```

Comment:
Can currently press the button multiple times for the same comment, which results in duplicates

There's no feedback to the user if their user is not on the database, is there a way around this?